provider/maas/constraints_test.go

```diff
@@ -202,87 +204,6 @@ func stringslicep(values ...string) *[]string {
 	return &values
 }
 
-func (suite *environSuite) TestSelectNodeValidZone(c *gc.C) {
-	env := suite.makeEnviron()
```

Comment:
Any test with this line was *always* testing against a MAAS with version 1.0 of the API.